src/backend/distributed/executor/multi_utility.c

```diff
@@ -300,7 +306,7 @@ VerifyTransmitStmt(CopyStmt *copyStatement)
  * COPYing from distributed tables and preventing unsupported actions.
  */
```

Comment:
Maybe add a note about a NULL return meaning that further processing should be skipped?